fundamentals/promises.md

```diff
@@ -70,44 +81,61 @@ If you are only interested in the error case, you can pass `null` for the first
 somePromise.then(null, onRejected)
 ```
 
-or you can use a second method exposed by a promise object, which is just a short-hand for calling `then()` with `null` as its first argument:
+or you can use a second method exposed by a promise, which is just a short-hand for calling `then()` with `null` as its first argument:
```

Comment:
Good point here. I also find this line a bit confusing, especially if you didn't know the details of how `then()` works, might be worth further clarifying how `catch()` is used in combination with `then()`.

Reply:
I've added a `.then()` before the `.catch()` in the code snippet. Further down there is an explanation why `.catch()` should be at the end of the chain, but I realise at this point in the text it remains a bit vague.